controlplane/src/core/repositories/OrganizationRepository.ts

```diff
@@ -63,7 +67,7 @@ export class OrganizationRepository {
       id: insertedOrg[0].id,
       name: insertedOrg[0].name,
       slug: insertedOrg[0].slug,
-      creatorUserId: insertedOrg[0].createdBy,
+      creatorUserId: insertedOrg[0].createdBy || undefined,
```

Comment:
When `createdBy` can be null by design, why is it required to fallback to undefined?

Reply:
It is how the type is defined in the DTO.

Reply:
ok